src/app/core/components/widgets/widgetpool/widgetpool.component.html

```diff
@@ -108,7 +108,7 @@ <h4>{{"Data not available" | translate}} <span *ngIf="!poolState.is_decrypted"><
                     <strong>{{"Available Space" | translate}}: </strong>&nbsp;&nbsp; 
                     <span *ngIf="volumeData && volumeData.used_pct;else unavail">
                       <span class="value">{{diskSize}}</span>
-                      <span class="value-units">{{diskSizeLabel}}</span>&nbsp;&nbsp;
+                      <span class="value-units"> {{diskSizeLabel}}</span>&nbsp;&nbsp;
```

Comment:
this space doesn't appear to show through for me, I am still seeing it crammed together on mine